pennylane/qchem/convert_openfermion.py

```diff
@@ -0,0 +1,168 @@
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires openfermion. "
+            "It can be installed with: pip install openfermion."
+        ) from Error
+
+    return openfermion
+
+
+def from_openfermion(ops, tol=1.0e-16, **kwargs):
+    r"""Convert OpenFermion ``QubitOperator`` to a :class:`~.LinearCombination` object in PennyLane representing a linear combination of qubit operators.
+
+    Args:
+        qubit_operator (QubitOperator): fermionic-to-qubit transformed operator in terms of
+            Pauli matrices
+        wires (Wires, list, tuple, dict): Custom wire mapping used to convert the qubit operator
+            to an observable terms measurable in a PennyLane ansatz.
+            For types Wires/list/tuple, each item in the iterable represents a wire label
+            corresponding to the qubit number equal to its index.
+            For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted.
+            If None, will use identity map (e.g. 0->0, 1->1, ...).
+        tol (float): tolerance value to decide whether the imaginary part of the coefficients is retained
+        return_sum (bool): flag indicating whether a ``Sum`` object is returned
+
+    Returns:
+        (pennylane.ops.Sum, pennylane.ops.LinearCombination): a linear combination of Pauli words
+
+    **Example**
+
+    >>> q_op = QubitOperator('X0', 1.2) + QubitOperator('Z1', 2.4)
+    >>> q_op
+    1.2 [X0] +
+    2.4 [Z1]
+    >>> from_openfermion(q_op)
+    1.2 * X(0) + 2.4 * Z(1)
+    """
+
+    coeffs, pl_ops = _openfermion_to_pennylane(ops, tol=tol)
+    pl_term = qml.ops.LinearCombination(coeffs, pl_ops)
+
+    if "format" in kwargs:
+        if kwargs["format"] == "Sum":
+            return qml.dot(*pl_term.terms())
+        if kwargs["format"] != "LinearCombination":
+            f = kwargs["format"]
+            raise ValueError(f"format must be a Sum or LinearCombination, got: {f}.")
+
+    return pl_term
+
+
+def to_openfermion(
+    pl_op: Union[Sum, LinearCombination, FermiWord, FermiSentence], wires=None, tol=1.0e-16
+):
+    r"""Convert a PennyLane operator to a OpenFermion ``QubitOperator`` or ``FermionOperator``.
+
+    Args:
+        pl_op (pennylane.ops.Sum, pennylane.ops.LinearCombination, pennylane.fermi.FermiWord, pennylane.fermi.FermiSentence):
+            linear combination of operators
+        wires (Wires, list, tuple, dict):
+            Custom wire mapping used to convert the qubit operator
+            to an observable terms measurable in a PennyLane ansatz.
+            For types Wires/list/tuple, each item in the iterable represents a wire label
+            corresponding to the qubit number equal to its index.
+            For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted.
+            If None, will use identity map (e.g. 0->0, 1->1, ...).
+
+    Returns:
+        (QubitOperator, FermionOperator): an OpenFermion operator
+
+    **Example**
```

Comment:
I could not reproduce this example. Could you please correct it? Also could not remove the imaginary part of the coefficients.